notebooks/210-ct-scan-live-inference/210-ct-scan-live-inference.ipynb

```diff
@@ -116,9 +113,11 @@
    "source": [
     "## Download and Prepare Data\n",
     "\n",
-    "Download one validation video for live inference. We reuse the KitsDataset class that was also used in the training and quantization notebook that will be released later.\n",
+    "Download one validation video for live inference. \n",
+    "\n",
+    "This tutorial reuses the `KitsDataset` class that was also used in the training and quantization notebook that will be released later.\n",
     "\n",
-    "Data is expected in `BASEDIR` defined in the Settings cell. `BASEDIR` should contain directories named `case_00000` to `case_00299`. If data for the case specified above does not exist yet, it will be downloaded and extracted in the next cell."
+    "The data is expected in the `BASEDIR` directory. The `BASEDIR` should contain the `case_00000` to `case_00299` subdirectories. If the data for the case specified above does not already exist, it will be downloaded and extracted in the next cell."
```

Comment:
```suggestion
    "The data is expected in `BASEDIR`. The `BASEDIR` directory should contain the `case_00000` to `case_00299` subdirectories. If the data for the case specified above does not already exist, it will be downloaded and extracted in the next cell."
```